src/components/TopBar/TopBarStyles.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+const bottomLineHeight = 1;
+const topBarHeight = 56;
+
+export const StyledTopBarWrapper = styled.nav`
+  position: absolute;
```

Comment:
Why do we need a wrapper? :)

Can't we just put the bottom line inside the TopBar?

```
<Header>
  <Logo />
  <Nav />
  <BottomLine />
</Header>
```

Which benefits this wrapper brings us?
Which problem does it solve?